rest/model/host.go

```diff
@@ -117,18 +118,20 @@ func (apiHost *APIHost) buildFromHostStruct(h interface{}) error {
 	apiHost.AvailabilityZone = ToStringPtr(v.Zone)
 	apiHost.DisplayName = ToStringPtr(v.DisplayName)
 	apiHost.HomeVolumeID = ToStringPtr(v.HomeVolumeID)
+	apiHost.LastCommunicationTime = v.LastCommunicationTime
 	apiHost.TotalIdleTime = NewAPIDuration(v.TotalIdleTime)
 	apiHost.CreationTime = ToTimePtr(v.CreationTime)
 
-	imageId, err := v.Distro.GetImageID()
-	if err != nil {
-		// report error but do not fail function because of a bad imageId
-		errors.Wrap(err, "problem getting image ID")
-	}
 	di := DistroInfo{
 		Id:       ToStringPtr(v.Distro.Id),
 		Provider: ToStringPtr(v.Distro.Provider),
-		ImageId:  ToStringPtr(imageId),
+	}
+	if !graphQL {
```

Comment:
a better choice of name for this variable would be to explicitly state the purpose of the boolean, which is to include or to not include the imageId. My suggestion would be `includeImageID`

Reply:
and make it true by default

Reply:
I agree, includeImageID is a better name for the boolean. I'm not aware of how to make a boolean true by default in a go function (I know it's possible in python). Is it possible to do?